README.md

```diff
@@ -55,7 +58,7 @@ Following command installs Hazelcast Go client:
 ```
 go get github.com/hazelcast/hazelcast-go-client
 ```
-[For more details](https://github.com/hazelcast/hazelcast-go-client/tree/master/samples/hello-world/README.md)
+[For more details](https://github.com/hazelcast/hazelcast-go-client/tree/master/sample/hello-world/README.md)
```

Comment:
link is still broken. `hello-world` -> `helloworld`
and no need to direct to `README.md`. It is already conveniently readable when they go to https://github.com/hazelcast/hazelcast-go-client/tree/master/sample/helloworld